nbodykit/cosmology/cosmology.py

```diff
@@ -10,6 +10,8 @@ class Cosmology(object):
     A cosmology calculator based on the CLASS binding in :mod:`classylss`.
 
     It is a collection of all method provided by the CLASS interfaces.
+    The object is immutable. To obtain an instance with a new set of parameters
+    use :method:`clone` or :method:`match`.
```

Comment:
Should we add an explicit __setattr__ that raises an Exception? I think you can currently set attributes, although it doesn't do anything.